stdlib/socket/src/main/ballerina/socket/udp_client_socket.bal

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Represents UDP socket client and related remote functions.
+#
+# + localPort - the local port number to which this socket is bound
+# + localAddress - the local IP address string in textual presentation to which the socket is bound
+# + id - a unique identifier to identify each client
+public type UdpClient client object {
+
+    private UdpClientConfig config;
+    public int localPort = 0;
+    public string? localAddress = ();
+    public int id = 0;
+
+    public function __init(UdpClientConfig? clientConfig) {
```

Comment:
Should this be a deaultable param? 
```suggestion
    public function __init(UdpClientConfig? clientConfig = ()) {
```

That way the user would not have to give an empty record literal like in https://github.com/ballerina-platform/ballerina-lang/pull/14364/files#diff-925cdfc5930b3e9edcc7a9e998fda05bR11 when initiating the client, if he doesn't want to specify anything additional 
```ballerina
    socket:UdpClient socketClient = new();
```